contract/vm_test.go

```diff
@@ -8,13 +8,22 @@ import (
 	"io/ioutil"
 	"math/big"
 	"os"
+	"runtime"
 	"strconv"
 	"strings"
 	"testing"
 
 	"github.com/aergoio/aergo/types"
 )
 
+func TestMain(m *testing.M) {
+	if runtime.GOARCH != "amd64" {
+		fmt.Println("skip architecture dependent test")
```

Comment:
It is just printing this, without command to skip
Is this working?

Reply:
The code `m.Run()` do actual test, so no test will be run and I think it's actually same as skipping test. But it can show different message in `t.Skip()`. Is this can make a side effect?

Reply:
Oh, I understood. So there is no need to change it